plugins/generator-1.18.2/forge-1.18.2/painting.definition.yaml

```diff
@@ -1,3 +1,5 @@
 global_templates:
   - template: elementinits/paintings.java.ftl
-    name: "@SRCROOT/@BASEPACKAGEPATH/init/@JavaModNamePaintings.java"
\ No newline at end of file
+    name: "@SRCROOT/@BASEPACKAGEPATH/init/@JavaModNamePaintings.java"
+
+field_exclusions: [title, author]
```

Comment:
Those exclusions should be omitted as now help tips cover this. Otherwise one will not be able to set author and go past validation in eg. 1.18.2

Reply:
Same for 1.19.2

Reply:
Done!